src/content/ui-api/kubectl-gs/get-appcatalogs.md

```diff
@@ -0,0 +1,112 @@
+---
+linkTitle: get appcatalogs
+title: "'kubectl gs get appcatalogs' command reference"
+description: Reference documentation on how to list appcatalogs and get details for a single appcatalog using 'kubectl gs'.
+weight: 20
+menu:
+  main:
+    parent: uiapi-kubectlgs
+aliases:
+  - /reference/kubectl-gs/get-appcatalogs/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+user_questions:
+  - How can I list appcatalogs using kubectl?
+  - How can I inspect appcatalogs using kubectl?
+---
+
+# `kubectl gs get appcatalogs`
+
+Like with all `get` commands in `kubectl`, this command can be used to get details on one item, an [AppCatalog]({{< relref "/ui-api/management-api/crd/appcatalogs.application.giantswarm.io.md" >}})
+custom resource in this case, or list several of them.
+
+## Usage
+
+### Get a list of appcatalogs {#list}
```

Comment:
What do you think about having `App Catalogs` instead of `appcatalogs` everywhere? It would be consistent with our other docs

```suggestion
### Get a list of App Catalogs {#list}
```

Reply:
Thanks, yes I think it's better with that casing.